docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,231 @@
+
+
+Site changes
+************
+
+When updating a site configuration from Gluon 2018.2.x, the following changes
+must be made:
+
+site.mk
+=======
+
+- We now require the ``GLUON_DEPRECATED`` variable to be set to decide how to handle the image generation for deprecated devices. (`#1745 <https://github.com/freifunk-gluon/gluon/pull/1745>`_)
+
+site.conf
+=========
+
+- The options ``wifi*.basic_rates`` and ``wifi*.supported_rates`` have been removed, as the legacy 802.11b rates are now disabled by default. (`#1716 <https://github.com/freifunk-gluon/gluon/pull/1716>`_)
+
+
+Internals
+*********
+
+BATMAN-adv multicast optimizations
+==================================
+
+The BATMAN-adv routing protocol has its multicast optimizations reactivated. This should further reduce the background noise as for example ICMPv6 Neighbor Discovery is now directed to nodes instead instead of being broadcasted.
+
+
+lua target files
+================
+
+Target definitions were rewritten in Lua, this was required due to the need to deprecate devices. It also offers the option for more flexible tagging of devices in the future. (`#1745 <https://github.com/freifunk-gluon/gluon/pull/1745>`_)
+
+luacheck
+========
+
+Lua scripts can now be properly linted and analyzed using luacheck. Run ``luacheck package scripts target`` in the Gluon project root. (`#1741 <https://github.com/freifunk-gluon/gluon/pull/1741>`_)
+
+
+Docker build-environment
+========================
+
+A minimal docker-based build environment is now available in ``contrib/Dockerfile``. (`#1738 <https://github.com/freifunk-gluon/gluon/pull/1738>`_)
+
+
+Reload of domain related services
+=================================
+
+A mechanism to reload domain related services is now available. (`#1710 <https://github.com/freifunk-gluon/gluon/pull/1710>`_)
+
+
+.. _releases-v2019.1-known-issues:
+
+Known issues
+************
+
+* Default TX power on many Ubiquiti devices is too high, correct offsets are unknown (`#94 <https://github.com/freifunk-gluon/gluon/issues/94>`_)
+
+  Reducing the TX power in the Advanced Settings is recommended.
+
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is disabled (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promiscuous mode is disallowed).
+
+* Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+* Frequent reboots due to out-of-memory or high load due to memory pressure on weak hardware specially in larger meshes
```

Comment:
"specially" -> like in v2018.2.3 release notes, should be changed to "especially"

Reply:
Agreed.